client/src/components/ItemSummary.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import $ from 'jquery';
+import NameAndPrice from './NameAndPrice.jsx';
+import Description from './Description.jsx';
+
+class ItemSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      item: [],
+    };
+    this.getFirstItem = this.getFirstItem.bind(this);
```

Comment:
Why are we binding getFirstItem but not getItemById?

Reply:
I ended up getting rid of getFirstItem in favor of getItemById, getFirstItem was pretty much a placeholder.